2-mandatory/3-financial-times.js

```diff
@@ -14,7 +21,12 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    
+  return allArticleTitles.reduce((shortest, sentence) {
```

Comment:
Good idea to use the reduce function here - but you're missing the `=>` between the parameters and the body of the function.
Now that we've looked at arrow functions in the Week 4 lesson - maybe you can try fixing this?